<ui version="4.0" >
 <class>SipSettingsBase</class>
 <widget class="QWidget" name="SipSettingsBase" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>335</width>
    <height>399</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>VoIP</string>
  </property>
  <layout class="QVBoxLayout" >
   <property name="margin" >
    <number>0</number>
   </property>
   <property name="spacing" >
    <number>0</number>
   </property>
   <item>
    <widget class="QTabWidget" name="tabWidget" >
     <property name="currentIndex" >
      <number>2</number>
     </property>
     <widget class="QWidget" name="identity" >
      <attribute name="title" >
       <string>Identity</string>
      </attribute>
      <layout class="QVBoxLayout" >
       <property name="margin" >
        <number>9</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item>
        <layout class="QHBoxLayout" >
         <property name="margin" >
          <number>0</number>
         </property>
         <property name="spacing" >
          <number>6</number>
         </property>
         <item>
          <layout class="QVBoxLayout" >
           <property name="margin" >
            <number>0</number>
           </property>
           <property name="spacing" >
            <number>6</number>
           </property>
           <item>
            <widget class="QLabel" name="label_2" >
             <property name="text" >
              <string>User ID:</string>
             </property>
             <property name="buddy" >
              <cstring>userUri</cstring>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_3" >
             <property name="text" >
              <string>Host:</string>
             </property>
             <property name="buddy" >
              <cstring>hostPart</cstring>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label" >
             <property name="text" >
              <string>Name:</string>
             </property>
             <property name="buddy" >
              <cstring>fullName</cstring>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_4" >
             <property name="text" >
              <string>Password:</string>
             </property>
             <property name="buddy" >
              <cstring>password</cstring>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" >
           <property name="margin" >
            <number>0</number>
           </property>
           <property name="spacing" >
            <number>6</number>
           </property>
           <item>
            <widget class="QLineEdit" name="userUri" />
           </item>
           <item>
            <widget class="QLineEdit" name="hostPart" />
           </item>
           <item>
            <widget class="QLineEdit" name="fullName" />
           </item>
           <item>
            <widget class="QLineEdit" name="password" >
             <property name="echoMode" >
              <enum>QLineEdit::PasswordEchoOnEdit</enum>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QCheckBox" name="autoRegister" >
         <property name="text" >
          <string>Auto Register</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer>
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="proxy" >
      <attribute name="title" >
       <string>Proxy</string>
      </attribute>
      <layout class="QVBoxLayout" >
       <property name="margin" >
        <number>9</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item>
        <layout class="QHBoxLayout" >
         <property name="margin" >
          <number>0</number>
         </property>
         <property name="spacing" >
          <number>6</number>
         </property>
         <item>
          <layout class="QVBoxLayout" >
           <property name="margin" >
            <number>0</number>
           </property>
           <property name="spacing" >
            <number>6</number>
           </property>
           <item>
            <widget class="QLabel" name="label_5" >
             <property name="text" >
              <string>Host:</string>
             </property>
             <property name="buddy" >
              <cstring>proxyHost</cstring>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_6" >
             <property name="text" >
              <string>Port:</string>
             </property>
             <property name="buddy" >
              <cstring>proxyPort</cstring>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_7" >
             <property name="text" >
              <string>User ID:</string>
             </property>
             <property name="buddy" >
              <cstring>proxyUserId</cstring>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" >
           <property name="margin" >
            <number>0</number>
           </property>
           <property name="spacing" >
            <number>6</number>
           </property>
           <item>
            <widget class="QLineEdit" name="proxyHost" />
           </item>
           <item>
            <widget class="QLineEdit" name="proxyPort" />
           </item>
           <item>
            <widget class="QLineEdit" name="proxyUserId" />
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <spacer>
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="advanced" >
      <attribute name="title" >
       <string>Advanced</string>
      </attribute>
      <layout class="QVBoxLayout" >
       <property name="margin" >
        <number>9</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item>
        <widget class="QLabel" name="label_9" >
         <property name="text" >
          <string>Expiry Times (mins):</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" >
         <property name="margin" >
          <number>0</number>
         </property>
         <property name="spacing" >
          <number>6</number>
         </property>
         <item>
          <layout class="QVBoxLayout" >
           <property name="margin" >
            <number>0</number>
           </property>
           <property name="spacing" >
            <number>6</number>
           </property>
           <item>
            <widget class="QLabel" name="label_8" >
             <property name="text" >
              <string>Registration:</string>
             </property>
             <property name="buddy" >
              <cstring>registrationExpiry</cstring>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_10" >
             <property name="text" >
              <string>Presence:</string>
             </property>
             <property name="buddy" >
              <cstring>presenceExpiry</cstring>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" >
           <property name="margin" >
            <number>0</number>
           </property>
           <property name="spacing" >
            <number>6</number>
           </property>
           <item>
            <widget class="QSpinBox" name="registrationExpiry" />
           </item>
           <item>
            <widget class="QSpinBox" name="presenceExpiry" />
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QLabel" name="label_11" >
         <property name="text" >
          <string>Preferred Codec:</string>
         </property>
         <property name="buddy" >
          <cstring>preferredCodec</cstring>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="preferredCodec" />
       </item>
       <item>
        <spacer>
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
